tests/phpunit/E2E/Extern/RestTest.php

```diff
@@ -47,8 +47,7 @@ protected function setUp() {
   }
 
   protected function getRestUrl() {
-    return CRM_Core_Resources::singleton()
-      ->getUrl('civicrm', 'extern/rest.php');
+    return CRM_Utils_System::url('civicrm/api/3/rest', NULL, TRUE, NULL, FALSE, TRUE);
```

Comment:
The RestTest should probably be more parameterized (data-provider based). The things is... `extern/rest.php` is going wired into a lot of external places, and (for existing deployments) I think it's important to continue testing it. The policy should be more like:

* For the `civicrm/api/3/rest` URL, test on all CMS's
* For the `extern/rest.php` URL, test on all CMS's *except* D8/D9.